website/content/docs/internals/telemetry/metrics/autosnap.mdx

```diff
@@ -0,0 +1,30 @@
+---
+layout: docs
+page_title: "Telemetry reference: Automated snapshot metrics"
+description: >-
+  Technical reference for automated Raft snapshot telemetry values.
+---
+
+# Automated snapshots telemetry
+
+<EnterpriseAlert product="vault" />
+
+Automated snapshot telemetry provides information on snapshot activity for the
```

Comment:
```suggestion
Automated snapshot telemetry provides information on automatically scheduled snapshot activity for the
```

We should be careful to distinguish automated snapshots from snapshots in general.

Reply:
good point.